cluster-scope/overlays/nerc-ocp-prod/kustomization.yaml

```diff
@@ -6,56 +6,47 @@ commonLabels:
 resources:
 - ../common
 - ../../bundles/acct-mgt
-- ../../bundles/metallb
 - ../../bundles/patch-operator
 - ../../bundles/xdmod-reader
 - feature/odf
-- feature/external-ingress
 - ../../base/core/namespaces/openshift-gitops
 - externalsecrets
 - apiserver/cluster.yaml
 - secretstores
+- issuers
 - machineconfigs/configure-bond0
-- machineconfigs/set-rpfilter-loose
-- machineconfigs/public-ingress
 - nodenetworkconfigurationpolicies
-- metallb
 - clusterversion.yaml
+- certificates
 
 generatorOptions:
   disableNameSuffixHash: true
 
 configMapGenerator:
-  - name: cluster-monitoring-config
-    namespace: openshift-monitoring
-    files:
-      - config.yaml=configmaps/cluster-monitoring-config.yaml
+- files:
+  - config.yaml=configmaps/cluster-monitoring-config.yaml
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
 
 patches:
-- path: ingresscontrollers/default_patch.yaml
 - path: oauths/cluster_patch.yaml
-
-- target:
-    group: "metallb.io"
-    name: ".*"
-  path: patches/skipdryrunonmissingresource.yaml
-
-- target:
-    group: "external-secrets.io"
-    name: ".*"
-  path: patches/skipdryrunonmissingresource.yaml
-
-- target:
-    group: "operator.external-secrets.io"
-    name: ".*"
-  path: patches/skipdryrunonmissingresource.yaml
-
-- target:
-    group: "nmstate.io"
-    name: ".*"
-  path: patches/skipdryrunonmissingresource.yaml
-
-- target:
-    group: "redhatcop.redhat.io"
-    name: ".*"
-  path: patches/skipdryrunonmissingresource.yaml
+- path: patches/skipdryrunonmissingresource.yaml
+  target:
+    group: metallb.io
```

Comment:
Likely harmless but if we're removing metallb do we need this patch?

Reply:
It is harmless, it is not necessary...but I guess I'm inclined to leave it here in case we re-introduce metallb at some point.